packages/nitro-protocol/gas-benchmarks/vanillaSetup.ts

```diff
@@ -21,10 +24,10 @@ declare global {
   }
 }
 
-export let ethAssetHolder: Contract;
-export let erc20AssetHolder: Contract;
-export let nitroAdjudicator: Contract;
-export let token: Contract;
+export let ethAssetHolder: ETHAssetHolder & Contract;
```

Comment:
I don't think this intersection type is the canonical way to use typechain. But it works well!